src/varyDSL.js

```diff
@@ -98,11 +98,7 @@ VaryDSL.prototype._validate = function() {
 };
 
 VaryDSL.prototype._getSplit = function() {
-  if (this._splitRegistry) {
-    return this._splitRegistry[this._assignment.getSplitName()];
-  } else {
-    return null;
-  }
+  return this._splitRegistry.getSplit(this._assignment.getSplitName());
```

Comment:
this will return `undefined` instead of null if the registry is offline or if the split with that name doesn't exist. it should behave the same.